src/coreclr/vm/codeman.h

```diff
@@ -2662,4 +2662,22 @@ class EECodeInfo
 
 void ThrowOutOfMemoryWithinRange();
 
+// Represents information about an XARCH CPU
```

Comment:
Can this be in defined next to the place that this is used in the .cpp file? It does not need to be defined globally.

Reply:
Moved it into the cpp file.